exporter/signalfxexporter/internal/translation/constants.go

```diff
@@ -3,10 +3,644 @@
 
 package translation // import "github.com/open-telemetry/opentelemetry-collector-contrib/exporter/signalfxexporter/internal/translation"
 
-// DefaultTranslationRulesYaml defines default translation rules that will be applied to metrics if
-// config.TranslationRules not specified explicitly.
-// Keep it in YAML format to be able to easily copy and paste it in config if modifications needed.
-const DefaultTranslationRulesYaml = `
+var DefaultTranslationRules = []Rule{
```

Comment:
Why do we need to change it from yaml? The default rules need to be easy read

Reply:
I see the yaml is still there, but it's harder to find out. I don't see a reason to have this hardcoded translation

Reply:
That's fair. The original implementation of parsing YAML into the `config` struct can't be followed exactly the same anymore, with the removal of the `translation_rules` config option, so I thought it would make sense to move away from the YAML parsing logic altogether. I can see that YAML is easier to read and modify, if/when necessary, than what I've proposed here, I just thought it made more sense than having to keep the original logic.

Reply:
_Updated._